assets/component-pickup-availability.css

```diff
@@ -55,6 +58,17 @@ pickup-availability-drawer {
     transform var(--duration-default) ease;
   transform: translateX(100%);
   width: 100%;
+  border-width: 0 0 0 var(--popup-drawer-border-width);
```

Comment:
Same here. With the border radius, it does the same as for the menu drawer.

Reply:
Same as above: https://github.com/Shopify/dawn/pull/865#issuecomment-969344919